src/components/main/index.tsx

```diff
@@ -1,7 +1,28 @@
 import React from "react";
+import { Link } from "react-router-dom";
+
+import { clearLocalStorage, getLocalStorage, urlFormat } from "@/utils";
+import { Pages } from "@/pages";
+import { logoutRequest } from "@/services";
 
 const Main = () => {
-  return <div>Main</div>;
+  const user = getLocalStorage("user");
```

Comment:
Maybe it's better to define keys storage like "user" in one place and use it only from there to avoid any error.

Reply:
Perfect, I'll add an enum as I did with the pages